app/views/user_mailer/other_submission_rejected.erb

```diff
@@ -13,9 +13,9 @@
           <td class='email-sub-title'>
 
             <p>Dear <%= @submission.name %>,</p>
-            <p>Thank you for your interest in selling with <%= link_to 'Artsy', 'https://www.artsy.net', target: :_blank %> and for giving us a chance to consider your artwork.</p>
+            <p>Thank you for your interest in selling with <%= link_to 'Artsy', artsy_formatted_url('', @utm_params), target: :_blank %> and for giving us a chance to consider your artwork.</p>
             <p>Our specialists have reviewed your submission and while it’s a wonderful piece, we unfortunately don’t have demand for this artwork on Artsy at the moment.</p>
-            <p>If you’d like to know more, you can read about what our specialists are looking for in our <%= link_to 'Collector Help Center', 'https://support.artsy.net/hc/en-us/categories/360003689533-Sell', target: :_blank %></p>
+            <p>If you’d like to know more, you can read about what our specialists are looking for in our <%= link_to 'Collector Help Center', utm_url('https://support.artsy.net/hc/en-us/categories/360003689533-Sell', @utm_params), target: :_blank %></p>
```

Comment:
This function adds the UTM params to the support.artsy.net link, should we keep it?